tex/LaTeXML/Portfile

```diff
@@ -17,7 +16,7 @@ long_description \
 
 # Written in Perl, but it is an application, not just modules
 PortGroup           perl5 1.0
-perl5.branches      5.28
+perl5.branches      5.34
```

Comment:
I think this should ideally match the default Perl version that MacPorts is using (i.e., the one in the `perl5` PG), which is currently still 5.28

Reply:
Ah, ok. I see the default is still p5.28  (I had seen a PR updating to 5.34, but I guess that only added a version for Perl Modules).

@reneeotten Thanks; I've updated the Portfile (force push to squash the commits)